NEWS

```diff
@@ -26,6 +26,10 @@ Several important changes and enancements were added:
      cat /proc/self/maps | grep '\[vvar]'
 * Support for glibc version 2.21 added.  To see if this affects you, do:
      ls -l /lib*/libc.so.6 /lib/*/libc.so.6
+* The environment variable DMTCP_GDB_ATTACH_ON_RESTART was added.  Setting
```

Comment:
I had meant to add something like the following to the print statement:
```Setting this can be a security risk.  But on a temporary basis, it can enable
easier debugging of restarted processes:```
[ I agree that we should also add this to:  `DMTCP_ROOT/doc/debugging-dmtcp.txt` .  I'll include the revised print statement and modified `debugging-dmtcp.txt` in the next iteration.